evm/test/IntegrationStandalone.t.sol

```diff
@@ -209,6 +219,19 @@ contract TestEndToEndBase is Test, IRateLimiterEvents {
             );
         }
 
+        // Get the TransferRedeemed(bytes32) event to ensure it matches up with the TransferSent(bytes32) event earlier
+        recordedLogs = vm.getRecordedLogs();
+        bytes32 recvEventDigest;
+        for (uint256 i = 0; i < recordedLogs.length; i++) {
+            if (recordedLogs[i].topics[0] == keccak256("TransferRedeemed(bytes32)")) {
+                recvEventDigest = recordedLogs[i].topics[1];
```

Comment:
Should there be a `break` after this assignment?

Reply:
fair. only in a test, so will cover in #522 if pushing again